src/mds/MDBalancer.cc

```diff
@@ -219,18 +243,16 @@ void MDBalancer::handle_export_pins(void)
 
 void MDBalancer::tick()
 {
-  static int num_bal_times = g_conf()->mds_bal_max;
-  auto bal_interval = g_conf().get_val<int64_t>("mds_bal_interval");
-  auto bal_max_until = g_conf().get_val<int64_t>("mds_bal_max_until");
+  static int num_bal_times = g_conf().get_val<int64_t>("mds_bal_max");
```

Comment:
We should not have static mutable variables where possible. Please move this to `MDBalancer`.

Reply:
Ack. I was wondering why this alone was initialized as static and so left it untouched.

Reply:
^ ?